crates/win-api-wrappers/src/wts.rs

```diff
@@ -11,6 +15,70 @@ impl WTSVirtualChannel {
         Self(handle)
     }
 
+    pub fn open_dvc(name: &str) -> anyhow::Result<Self> {
+        let channel_name = AnsiString::from(name);
+
+        #[allow(clippy::undocumented_unsafe_blocks)] // false positive
```

Comment:
I recommend to use `#[expect]` so we are warned once the false positive is resolved :)